index.template.php

```diff
@@ -78,6 +79,7 @@
 	<script src="//play.pokemonshowdown.com/data/formats-data.js?"></script>
 	<script src="//play.pokemonshowdown.com/data/typechart.js?"></script>
 	<script src="//play.pokemonshowdown.com/data/aliases.js?"></script>
+	<script src="//play.pokemonshowdown.com/js/battle-dex-search.js?"></script>
```

Comment:
?

Reply:
Without including `battle-dex-search.js`, I get a ReferenceError that DexSearch is not defined in `search.js`.

Reply:
Did I forget to push this?

Reply:
Looks like yes, can you rebase on my new update?

Reply:
Done.